package.json

```diff
@@ -16,5 +16,8 @@
     },
     "lint-staged": {
         "**/*": "prettier --write --ignore-unknown"
+    },
+    "dependencies": {
```

Comment:
This isn't the right place to add this dependency, it needs to be in the `demo` workspace